SimGeneral/PreMixingModule/python/mixOne_premix_on_sim_cfi.py

```diff
@@ -319,10 +327,3 @@
 # Run-dependent MC
 from Configuration.ProcessModifiers.runDependent_cff import runDependent
 runDependent.toModify(mixData.workers.ecal, timeDependent=True)
```

Comment:
I am not entirely sure this is the right fix, but once we have a premixing library that allows to merge back `runDependent` and `runDependentForPixel` the whole section should be moved up. Pinging @cms-sw/alca-l2.

Reply:
I agree with this plan